src/main/java/seedu/mark/model/reminder/Note.java

```diff
@@ -0,0 +1,51 @@
+package seedu.mark.model.reminder;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.mark.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represent a note in reminder.
```

Comment:
```suggestion
 * Represents a note in reminder.
```

@kxyoke i saw your #46 has `Note` class as well? Can we change the name of either annotation note or reminder note so we wont have duplicate class names?

Reply:
Can, I'll name it `AnnotationNotes`